backend/app/schema.py

```diff
@@ -13,9 +13,7 @@ class User(TypedDict):
     """The time the user was created."""
 
 
-class Assistant(TypedDict):
-    """Assistant model."""
-
+class Assistant(BaseModel):
```

Comment:
Was this change necessary or just to make it more consisten w/ using pydantic for models throughout the app?

Reply:
Just to make it more consistent—same as with the User and Thread schemas. What do you think?

Reply:
Yep that works